supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc

```diff
@@ -42,39 +42,26 @@ Always use "Administration Portal", including the capital P. When other departme
 ==== image:images/yes.png[yes] Data Warehouse (noun)
 *Description*: The Manager includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
 
-Data Warehouse is optional in Red Hat Enterprise Virtualization (3.x). It is mandatory in Red Hat Virtualization (4.x).
+Data Warehouse is mandatory in Red Hat Virtualization.
 
 Always spell out in full and capitalize as shown here, unless part of a command. Use "Data Warehouse", not "the Data Warehouse", unless referring to "the Data Warehouse package", "the Data Warehouse service", etc.
```

Comment:
We could avoid this 'etc.' (IBM style no-no) by explaining the change of use of the term. Could do something like:

`Use "Data Warehouse", not "the Data Warehouse", unless "Data Warehouse" functions as an adjective, for example, "the Data Warehouse package" or "the Data Warehouse service".`

 The 'for example' would remove the need for an etc.' or an 'and so on' too.